doc generation is failing with sphinx 1.7

```
env PYTHONPATH=/build/morse-simulator-1.4/obj-x86_64-linux-gnu:/build/morse-simulator-1.4/obj-x86_64-linux-gnu/src:/build/morse-simulator-1.4/obj-x86_64-linux-gnu/fakeenv:/build/morse-simulator-1.4/src:/build/morse-simulator-1.4/testing:/build/morse-simulator-1.4/bindings/pymorse/src/:$PYTHONPATH PYTHONDONTWRITEBYTECODE="morse" MORSESOURCE=/build/morse-simulator-1.4 /usr/bin/python3 /usr/bin/sphinx-build -b man -c /build/morse-simulator-1.4/obj-x86_64-linux-gnu/doc /build/morse-simulator-1.4/doc/man /build/morse-simulator-1.4/obj-x86_64-linux-gnu/doc/man && /bin/gzip -f /build/morse-simulator-1.4/obj-x86_64-linux-gnu/doc/man/*.1
Running Sphinx v1.7.2

Extension error:
Could not import extension vimeo (exception: cannot import name 'Directive')
make[4]: *** [CMakeFiles/man.dir/build.make:60: CMakeFiles/man] Error 2
make[4]: Leaving directory '/build/morse-simulator-1.4/obj-x86_64-linux-gnu'
```

--- a/doc/conf.py.in
+++ b/doc/conf.py.in
@@ -26,7 +26,7 @@ sys.path.append('@CMAKE_CURRENT_SOURCE_DIR@/doc/exts')
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo', 'sphinx.ext.viewcode', 'vimeo','gallery', 'tag', 'examples']
-extensions.append('sphinx.ext.pngmath')
+extensions.append('sphinx.ext.imgmath')
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['@CMAKE_CURRENT_SOURCE_DIR@/doc/morse-theme/templates', '@CMAKE_CURRENT_SOURCE_DIR@/doc/sphinxext']

--- a/doc/exts/gallery.py
+++ b/doc/exts/gallery.py
@@ -4,8 +4,7 @@
 import os
 import glob
 from docutils import nodes
-from docutils.parsers.rst import directives
-from sphinx.util.compat import Directive
+from docutils.parsers.rst import directives, Directive
 
 IMAGE_PATH="_images/"
 HTML_PATH="user/"

--- a/doc/exts/vimeo.py
+++ b/doc/exts/vimeo.py
@@ -5,8 +5,7 @@ from __future__ import division
 
 import re
 from docutils import nodes
-from docutils.parsers.rst import directives
-from sphinx.util.compat import Directive
+from docutils.parsers.rst import directives, Directive
 
 CONTROL_HEIGHT = 30
 

